[2025-02-06T11:38:47.318+0000] {processor.py:186} INFO - Started process (PID=74) to work on /root/airflow/dags/training_dag.py
[2025-02-06T11:38:47.321+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T11:38:47.327+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:38:47.326+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T11:38:48.472+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T11:38:48.963+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:38:48.962+0000] {override.py:1912} INFO - Created Permission View: can edit on DAG:my_training_dag
[2025-02-06T11:38:49.212+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:38:49.211+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG:my_training_dag
[2025-02-06T11:38:49.267+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:38:49.266+0000] {override.py:1912} INFO - Created Permission View: can read on DAG:my_training_dag
[2025-02-06T11:38:49.349+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:38:49.349+0000] {override.py:1912} INFO - Created Permission View: menu access on DAG Run:my_training_dag
[2025-02-06T11:38:49.406+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:38:49.405+0000] {override.py:1912} INFO - Created Permission View: can create on DAG Run:my_training_dag
[2025-02-06T11:38:49.461+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:38:49.460+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG Run:my_training_dag
[2025-02-06T11:38:49.531+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:38:49.531+0000] {override.py:1912} INFO - Created Permission View: can read on DAG Run:my_training_dag
[2025-02-06T11:38:49.532+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:38:49.532+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T11:38:49.559+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:38:49.558+0000] {dag.py:3262} INFO - Creating ORM DAG for my_training_dag
[2025-02-06T11:38:49.574+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:38:49.574+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to 2025-01-01 00:00:00+00:00, run_after=2025-01-01 00:00:30+00:00
[2025-02-06T11:38:49.675+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 2.365 seconds
[2025-02-06T11:43:10.928+0000] {processor.py:186} INFO - Started process (PID=1071) to work on /root/airflow/dags/training_dag.py
[2025-02-06T11:43:10.929+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T11:43:10.932+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:43:10.931+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T11:43:11.700+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T11:43:11.726+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:43:11.725+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T11:43:11.750+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:43:11.750+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to 2025-01-01 00:00:00+00:00, run_after=2025-01-01 00:00:30+00:00
[2025-02-06T11:43:11.821+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.900 seconds
[2025-02-06T11:47:37.483+0000] {processor.py:186} INFO - Started process (PID=1965) to work on /root/airflow/dags/training_dag.py
[2025-02-06T11:47:37.487+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T11:47:37.495+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:47:37.494+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T11:47:39.407+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:47:39.393+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from functions import *
ModuleNotFoundError: No module named 'functions'
[2025-02-06T11:47:39.413+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T11:47:39.499+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 2.033 seconds
[2025-02-06T11:51:14.120+0000] {processor.py:186} INFO - Started process (PID=2939) to work on /root/airflow/dags/training_dag.py
[2025-02-06T11:51:14.122+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T11:51:14.123+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:51:14.123+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T11:51:14.862+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:51:14.856+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from functions import *
ModuleNotFoundError: No module named 'functions'
[2025-02-06T11:51:14.865+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T11:51:14.922+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.807 seconds
[2025-02-06T11:54:38.240+0000] {processor.py:186} INFO - Started process (PID=74) to work on /root/airflow/dags/training_dag.py
[2025-02-06T11:54:38.242+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T11:54:38.247+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:54:38.246+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T11:54:39.107+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:54:39.098+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from functions import *
ModuleNotFoundError: No module named 'functions'
[2025-02-06T11:54:39.111+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T11:54:39.191+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.959 seconds
[2025-02-06T11:55:28.219+0000] {processor.py:186} INFO - Started process (PID=74) to work on /root/airflow/dags/training_dag.py
[2025-02-06T11:55:28.222+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T11:55:28.227+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:55:28.226+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T11:55:29.287+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:55:29.277+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from functions import *
ModuleNotFoundError: No module named 'functions'
[2025-02-06T11:55:29.291+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T11:55:29.384+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 1.174 seconds
[2025-02-06T11:58:03.109+0000] {processor.py:186} INFO - Started process (PID=75) to work on /root/airflow/dags/training_dag.py
[2025-02-06T11:58:03.111+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T11:58:03.115+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:58:03.115+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T11:58:04.213+0000] {logging_mixin.py:190} INFO - [2025-02-06T11:58:04.195+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from functions import *
ModuleNotFoundError: No module named 'functions'
[2025-02-06T11:58:04.219+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T11:58:04.319+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 1.218 seconds
[2025-02-06T12:00:01.951+0000] {processor.py:186} INFO - Started process (PID=75) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:00:01.953+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:00:01.956+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:00:01.956+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:00:02.796+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:00:02.787+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from dags.utiles.functions import *
ModuleNotFoundError: No module named 'dags'
[2025-02-06T12:00:02.799+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:00:02.867+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.927 seconds
[2025-02-06T12:00:20.952+0000] {processor.py:186} INFO - Started process (PID=74) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:00:20.954+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:00:20.959+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:00:20.958+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:00:21.782+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:00:21.771+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from dags.utiles.functions import *
ModuleNotFoundError: No module named 'dags'
[2025-02-06T12:00:21.784+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:00:21.839+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.897 seconds
[2025-02-06T12:00:50.141+0000] {processor.py:186} INFO - Started process (PID=76) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:00:50.143+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:00:50.145+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:00:50.145+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:00:50.639+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:00:50.634+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from dags.utiles.functions import *
ModuleNotFoundError: No module named 'dags'
[2025-02-06T12:00:50.641+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:00:50.678+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.541 seconds
[2025-02-06T12:03:33.662+0000] {processor.py:186} INFO - Started process (PID=75) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:03:33.664+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:03:33.668+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:03:33.667+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:03:34.456+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:03:34.452+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from utiles.functions import *
ModuleNotFoundError: No module named 'utiles'
[2025-02-06T12:03:34.458+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:03:34.506+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.850 seconds
[2025-02-06T12:06:28.003+0000] {processor.py:186} INFO - Started process (PID=1110) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:06:28.005+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:06:28.007+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:06:28.006+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:06:28.545+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:06:28.541+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from utiles.functions import *
ModuleNotFoundError: No module named 'utiles'
[2025-02-06T12:06:28.546+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:06:28.576+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.576 seconds
[2025-02-06T12:08:54.145+0000] {processor.py:186} INFO - Started process (PID=1961) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:08:54.146+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:08:54.148+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:08:54.147+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:08:54.974+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:08:54.968+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from utiles.functions import *
ModuleNotFoundError: No module named 'utiles'
[2025-02-06T12:08:54.977+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:08:55.020+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.880 seconds
[2025-02-06T12:11:06.027+0000] {processor.py:186} INFO - Started process (PID=81) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:11:06.028+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:11:06.030+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:11:06.030+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:11:06.490+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:11:06.485+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from utiles.functions import *
ModuleNotFoundError: No module named 'utiles'
[2025-02-06T12:11:06.491+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:11:06.538+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.516 seconds
[2025-02-06T12:14:04.672+0000] {processor.py:186} INFO - Started process (PID=75) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:14:04.673+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:14:04.676+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:14:04.676+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:14:05.190+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:14:05.185+0000] {dagbag.py:387} ERROR - Failed to import: /root/airflow/dags/training_dag.py
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/root/airflow/dags/training_dag.py", line 8, in <module>
    from utiles.functions import *
ModuleNotFoundError: No module named 'utiles'
[2025-02-06T12:14:05.192+0000] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:14:05.238+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.572 seconds
[2025-02-06T12:26:08.697+0000] {processor.py:186} INFO - Started process (PID=1851) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:26:08.698+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:26:08.700+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:26:08.699+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:26:09.272+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:26:09.362+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:26:09.362+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T12:26:09.375+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:26:09.375+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to 2025-01-01 00:00:00+00:00, run_after=2025-01-01 00:00:30+00:00
[2025-02-06T12:26:09.451+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.758 seconds
[2025-02-06T12:34:05.400+0000] {processor.py:186} INFO - Started process (PID=6020) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:34:05.402+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:34:05.403+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:34:05.403+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:34:05.928+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:34:05.948+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:34:05.947+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T12:34:05.967+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:34:05.967+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to 2025-01-01 00:35:00+00:00, run_after=2025-01-01 00:35:30+00:00
[2025-02-06T12:34:06.020+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.623 seconds
[2025-02-06T12:42:21.190+0000] {processor.py:186} INFO - Started process (PID=10243) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:42:21.191+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:42:21.193+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:42:21.193+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:42:21.788+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:42:21.823+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:42:21.823+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T12:42:21.857+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:42:21.857+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to 2025-01-01 01:10:30+00:00, run_after=2025-01-01 01:11:00+00:00
[2025-02-06T12:42:21.917+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.731 seconds
[2025-02-06T12:50:41.658+0000] {processor.py:186} INFO - Started process (PID=14201) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:50:41.659+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:50:41.661+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:50:41.661+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:50:42.236+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:50:42.255+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:50:42.254+0000] {taskinstance.py:3313} ERROR - {'DAG Id': 'my_training_dag', 'Task Id': 'task_load_data', 'Run Id': 'scheduled__2025-01-01T01:40:30+00:00', 'Hostname': 'f59ce29465b4'}
[2025-02-06T12:50:42.281+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2025-02-06T12:50:42.281+0000] {logging_mixin.py:190} INFO - Task start:2025-02-06 12:50:33.743774+00:00 end:2025-02-06 12:50:42.264187+00:00 duration:8.520413
[2025-02-06T12:50:42.282+0000] {logging_mixin.py:190} INFO - Task:<Task(PythonOperator): task_load_data> dag:<DAG: my_training_dag> dagrun:<DagRun my_training_dag @ 2025-01-01 01:40:30+00:00: scheduled__2025-01-01T01:40:30+00:00, state:running, queued_at: 2025-02-06 12:50:30.823120+00:00. externally triggered: False>
[2025-02-06T12:50:42.282+0000] {logging_mixin.py:190} INFO - Failure caused by {'DAG Id': 'my_training_dag', 'Task Id': 'task_load_data', 'Run Id': 'scheduled__2025-01-01T01:40:30+00:00', 'Hostname': 'f59ce29465b4'}
[2025-02-06T12:50:42.282+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:50:42.282+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=my_training_dag, task_id=task_load_data, run_id=scheduled__2025-01-01T01:40:30+00:00, execution_date=20250101T014030, start_date=20250206T125033, end_date=20250206T125042
[2025-02-06T12:50:42.322+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:50:42.321+0000] {processor.py:876} INFO - Executed callback for <TaskInstance: my_training_dag.task_load_data scheduled__2025-01-01T01:40:30+00:00 [failed]> in state failed
[2025-02-06T12:50:42.341+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:50:42.340+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T12:50:42.362+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:50:42.361+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to 2025-01-01 01:41:00+00:00, run_after=2025-01-01 01:41:30+00:00
[2025-02-06T12:50:42.423+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.770 seconds
[2025-02-06T12:52:44.695+0000] {processor.py:186} INFO - Started process (PID=15142) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:52:44.696+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:52:44.698+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:52:44.698+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:52:45.390+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:52:45.486+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:52:45.486+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T12:52:45.505+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:52:45.504+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to None, run_after=None
[2025-02-06T12:52:45.716+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 1.026 seconds
[2025-02-06T12:55:47.256+0000] {processor.py:186} INFO - Started process (PID=74) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:55:47.257+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:55:47.258+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:55:47.258+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:55:47.748+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:55:47.775+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:55:47.774+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T12:55:47.927+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:55:47.927+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to None, run_after=None
[2025-02-06T12:55:47.985+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.733 seconds
[2025-02-06T12:57:31.100+0000] {processor.py:186} INFO - Started process (PID=74) to work on /root/airflow/dags/training_dag.py
[2025-02-06T12:57:31.101+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T12:57:31.104+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:57:31.103+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T12:57:31.602+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T12:57:31.624+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:57:31.624+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T12:57:31.963+0000] {logging_mixin.py:190} INFO - [2025-02-06T12:57:31.962+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to None, run_after=None
[2025-02-06T12:57:32.195+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 1.100 seconds
[2025-02-06T13:01:20.916+0000] {processor.py:186} INFO - Started process (PID=1071) to work on /root/airflow/dags/training_dag.py
[2025-02-06T13:01:20.918+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T13:01:20.920+0000] {logging_mixin.py:190} INFO - [2025-02-06T13:01:20.919+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T13:01:21.489+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T13:01:21.508+0000] {logging_mixin.py:190} INFO - [2025-02-06T13:01:21.508+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T13:01:21.527+0000] {logging_mixin.py:190} INFO - [2025-02-06T13:01:21.527+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to None, run_after=None
[2025-02-06T13:01:21.575+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.662 seconds
[2025-02-06T13:03:55.510+0000] {processor.py:186} INFO - Started process (PID=1902) to work on /root/airflow/dags/training_dag.py
[2025-02-06T13:03:55.512+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T13:03:55.513+0000] {logging_mixin.py:190} INFO - [2025-02-06T13:03:55.513+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T13:03:56.119+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T13:03:56.137+0000] {logging_mixin.py:190} INFO - [2025-02-06T13:03:56.137+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T13:03:56.158+0000] {logging_mixin.py:190} INFO - [2025-02-06T13:03:56.158+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to None, run_after=None
[2025-02-06T13:03:56.217+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.710 seconds
[2025-02-06T13:06:13.896+0000] {processor.py:186} INFO - Started process (PID=2641) to work on /root/airflow/dags/training_dag.py
[2025-02-06T13:06:13.897+0000] {processor.py:914} INFO - Processing file /root/airflow/dags/training_dag.py for tasks to queue
[2025-02-06T13:06:13.902+0000] {logging_mixin.py:190} INFO - [2025-02-06T13:06:13.902+0000] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow/dags/training_dag.py
[2025-02-06T13:06:14.619+0000] {processor.py:925} INFO - DAG(s) 'my_training_dag' retrieved from /root/airflow/dags/training_dag.py
[2025-02-06T13:06:14.641+0000] {logging_mixin.py:190} INFO - [2025-02-06T13:06:14.640+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-06T13:06:14.662+0000] {logging_mixin.py:190} INFO - [2025-02-06T13:06:14.662+0000] {dag.py:4180} INFO - Setting next_dagrun for my_training_dag to None, run_after=None
[2025-02-06T13:06:14.730+0000] {processor.py:208} INFO - Processing /root/airflow/dags/training_dag.py took 0.839 seconds
